.lintrc

```diff
@@ -0,0 +1,2 @@
+[tools]
+linters = flake8
```

Comment:
Installing flake8 globally on the developer's machine also can catch a lot of these issues before committing. :P